styleguide/source/_patterns/03-organisms/resource-tabs.twig

```diff
@@ -0,0 +1,247 @@
+<div class="ama__resource-tabs">
+  <ul class="ama__resource-tabs__nav">
+    <li data-resource="resources">
+      <div class="ama__resource-tabs__nav__icon"></div>
+      <div class="ama__resource-tabs__nav__label">Resources</div>
+    </li>
+    <li data-resource="media" class="--is-active">
+      <div class="ama__resource-tabs__nav__icon"></div>
+      <div class="ama__resource-tabs__nav__label">Media</div>
+    </li>
+    <li data-resource="downloads">
+      <div class="ama__resource-tabs__nav__icon"></div>
+      <div class="ama__resource-tabs__nav__label">Downloads</div>
+    </li>
+    <li data-resource="related-links">
+      <div class="ama__resource-tabs__nav__icon"></div>
+      <div class="ama__resource-tabs__nav__label">Related Links</div>
+    </li>
+    <li data-resource="schedules">
+      <div class="ama__resource-tabs__nav__icon"></div>
+      <div class="ama__resource-tabs__nav__label">Schedules</div>
+    </li>
+    <li data-resource="tables">
+      <div class="ama__resource-tabs__nav__icon"></div>
+      <div class="ama__resource-tabs__nav__label">Tables</div>
+    </li>
+    <li data-resource="contacts">
+      <div class="ama__resource-tabs__nav__icon"></div>
+      <div class="ama__resource-tabs__nav__label">Contacts</div>
+    </li>
+  </ul>
+  <div class="ama__resource-tabs__content">
```

Comment:
This also should go into a loop:
{% for content in ama__resource-tabs__content %}
{% include "molecules/resource-content.twig" %}
{% endfor %}

Reply:
All hard coded text need to go into a json file